linux/ibus-keyman/build.sh

```diff
@@ -20,10 +20,15 @@ builder_describe \
   "install                   install artifacts" \
   "uninstall                 uninstall artifacts" \
   "@/core:arch" \
-  "--no-integration          don't run integration tests"
+  "--no-integration          don't run integration tests" \
+  "--report                  create coverage report" \
+  "--coverage                capture test coverage"
 
 builder_parse "$@"
 
+builder_describe_internal_dependency \
+  report:engine test:engine
+
```

Comment:
```suggestion
```

Now that `--coverage` is an option checked inside the test action, this can go away, right?

Reply:
Oops. Thanks!